extensions/iceberg/src/main/java/io/deephaven/iceberg/util/IcebergCatalogAdapter.java

```diff
@@ -44,6 +44,8 @@ public class IcebergCatalogAdapter {
     private final Catalog catalog;
     private final FileIO fileIO;
 
+    private final Map<String, String> properties = new HashMap<>();;
```

Comment:
This is currently unused. Why do we have it?

Reply:
The plan is to have this property map populate the S3Instructions object. But I can't figure out how to do this within the current project structure.